docs/changes/2776.feature.rst

```diff
@@ -0,0 +1 @@
+Add HDF5Merger to the pix stats tool.
```

Comment:
I would say this is an implementation detail. What does this give the user?

Reply:
Please check if rephrasing suits better.